### PR TITLE
Add integration test for trashbin

### PR DESCRIPTION
**Pull request by MorrisJobke**
* downstream of https://github.com/owncloud/core/pull/27069

### CODE REVIEW

File: build/integration/features/bootstrap/BasicStructure.php [hunk 1]
```diff
@@ -35,6 +35,7 @@
 trait BasicStructure {
 
 	use Auth;
+	use Trashbin;
```
Comment (nickvergessen) [inline build/integration/features/bootstrap/BasicStructure.php]: :(

**Review by nickvergessen — APPROVED** [review]
Test passes, fine by me. However I still think the functional tests need some structure one day.

**Review by rullzer — APPROVED** [review]
:see_no_evil:

**Comment by MorrisJobke** [comment]
> Test passes, fine by me. However I still think the functional tests need some structure one day.

No they don't :/

**Comment by MorrisJobke** [comment]
Reverted because the unit tests fail.

**Pull request merged**